features/support/send_sigint_during_bisect.rb

```diff
@@ -5,8 +5,8 @@ module RSpec::Core::Formatters
   BisectProgressFormatter = Class.new(remove_const :BisectProgressFormatter) do
     RSpec::Core::Formatters.register self
 
-    def bisect_round_finished(notification)
-      return super unless notification.round == 3
+    def bisect_round_started(notification)
```

Comment:
Yeah, I don't really see a way around this one - there are two exit conditions for a round now, and because it's recursive we can only really say a round "ends" when all of its sub-rounds end, which doesn't really help much. So I ended up with `bisect_round_started` and two exit conditions, `bisect_ignoring_ids` and `bisect_multiple_culprits_detected`.

I'll try to think of better names for these to make it clear they're a related set of notifications.